tests/strands/telemetry/test_tracer.py

```diff
@@ -681,3 +681,43 @@ def test_serialize_vs_json_dumps():
     custom_result = serialize({"text": japanese_text})
     assert japanese_text in custom_result
     assert "\\u" not in custom_result
+
+
+def test_init_with_no_env_or_param():
+    """Test initializing with neither environment variable nor constructor parameter."""
+    with mock.patch.dict(os.environ, {}, clear=True):
```

Comment:
Let's mock the environment using a fixture like in https://github.com/strands-agents/tools/blob/main/tests/test_environment.py#L29

(you may not need `autouse=True`)